src/styles/utils/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --button-color-1: #fffffffe;
+  --button-color-2: #00acdc;
```

Comment:
it's better to name you variables like accent-color for blue color and primary-hover-color for its hover effect